### PR TITLE
Chore(Coverage): Integrate Coveralls Code coverage

### PR DESCRIPTION
**Pull request by alainmateso**
#### What does this PR do?
Integrate Coveralls code coverage
#### Description of Task to be completed?

- Add caret-bn-backend repo to coveralls
- Add .coveralls.yml file that contains the service name and repo token
- Add coveralls README badge

#### How should this be manually tested?

- Clone the repository from [here](https://github.com/andela/caret-bn-backend.git)
- Browse to the repo directory
- Run `npm install`
- Preview README file

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168781671
#### Screenshots (if appropriate)
Screen Shot 2019-09-30 at 12 33 29

#### Questions:
N/A

### CODE REVIEW

**Review by Cheza-Dzabala — COMMENTED** [review]
Please note that you did not add a label to your PR. ` WIP ` /  ` Ready for review ` as specified by the TTL in her slack communication.  
Screen Shot 2019-09-30 at 12 37 47

-- Notice the label.. All is in order

**Comment by alainmateso** [comment]
> Please note that you did not add a label to your PR. `WIP` / `Ready for review` as specified by the TTL in her slack communication.
> Screen Shot 2019-09-30 at 12 37 47

There is Ready for Review label on the PR

**Review by Cheza-Dzabala — APPROVED** [review]
All good!!

**Review by ngireric123 — APPROVED** [review]
it is now ok

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: tVZhMlTbg8dcMOfD8szWYSOgXigcwzoGs
```
Comment (nignanthomas) [inline .coveralls.yml]: I don't think it is a good idea to have your **repo_token** exposed
In addition, I think you don't need to add a .coveralls.yml here.
.coveralls.yml is usually for local use.
Kindly correct if I am wrong

**Review by nignanthomas — CHANGES_REQUESTED** [review]
You can add this file (.coveralls.yml)to .gitignore

File: .coveralls.yml (hunk 1 above)
Comment (alainmateso) [reply .coveralls.yml]: Done!

**Review by nignanthomas — APPROVED** [review]

**Review by nakiwuge — CHANGES_REQUESTED** [review]
@alainmateso good job however kindly add a body to your commit message

**Comment by alainmateso** [comment]
> @alainmateso good job however kindly add a body to your commit message

I have added the body. Let me know what you think of it!

**Review by nakiwuge — APPROVED** [review]

**Pull request merged**